pandas/io/formats/latex.py

```diff
@@ -657,7 +706,29 @@ def _select_builder(self) -> Type[TableBuilderAbstract]:
         return TabularBuilder
 
     @property
-    def column_format(self) -> str:
+    def caption(self) -> str:
```

Comment:
is there a reason we use setters / getters rather than just typing in teh class? seems much simpler.

Reply:
Seems that there is no particular reason as the class is supposed to be initialized only once. If this is so, then I will update this into a function, which is run at init.